### PR TITLE
Gotcha on Cloudfront HTTP Host headers

### PR DESCRIPTION
**Pull request by spudstuff**
Gotcha for CloudFront not forwarding HTTP Host headers by default.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1184,6 +1184,7 @@ CloudFront
 ### CloudFront Gotchas and Limitations
 
 -	If using S3 as a backing store, remember that the endpoints for website hosting and for general S3 are different. Example: “bucketname.s3.amazonaws.com” is a standard S3 serving endpoint, but to have redirect and error page support, you need to use the website hosting endpoint listed for that bucket, e.g. “bucketname.s3-website-us-east-1.amazonaws.com” (or the appropriate region).
+-   🔹By default, CloudFront will not forward HTTP Host: headers through to your origin servers. This can be problematic for your origin if you run multiple sites switched with host headers. You can [enable host header forwarding](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/RequestAndResponseBehaviorCustomOrigin.html#request-custom-headers-behavior) in the default cache behavior settings.
```
Comment (jlevy) [inline README.md]: Looks good. Can we make this the yellow diamond as that's what's appropriate here?

Comment (spudstuff) [inline README.md]: Sure @jlevy, do you mean the orange diamond? 🔸

Comment (spudstuff) [inline README.md]: Updated @jlevy.

**Comment by jlevy** [comment]
👍 Great, thanks!

**Pull request merged**